params/config.go

```diff
@@ -620,91 +629,107 @@ func (c *ChainConfig) CheckConfigForkOrder() error {
 		{name: "arrowGlacierBlock", block: c.ArrowGlacierBlock, optional: true},
 		{name: "grayGlacierBlock", block: c.GrayGlacierBlock, optional: true},
 		{name: "mergeNetsplitBlock", block: c.MergeNetsplitBlock, optional: true},
-		{name: "shanghaiBlock", block: c.ShanghaiBlock, optional: true},
 		{name: "cancunBlock", block: c.CancunBlock, optional: true},
+		{name: "shanghaiTime", timestamp: c.ShanghaiTime},
```

Comment:
Let's define Cancun as time and move it after the Shanghai.

Reply:
I'll do it in a followup PR, it will be a bit breaking and don't want to convolute it with this change.